open_rarity/scoring/scorer.py

```diff
@@ -17,6 +18,9 @@ class Scorer:
     def __init__(self) -> None:
         # OpenRarity uses InformationContent as the scoring algorithm of choice.
         self.handler = InformationContentScoringHandler()
+        # OpenRarity always normalizes individual traits based on the total number of
+        # possible values for an attribute name.
+        self.trait_normalized = True
```

Comment:
Can we not pass this flag to the scorer at all ? I think we have a default value for the normalized to be True.  Reason - for IC normalization has nothing to do with trait normalization. So Sum,Arithmetic, Geometric and Harmonic will take it as default param.

Reply:
I removed it because this is handler-specific and it doesn't make sense for IC handler.
In theory if we wanted to use a different handler you can just set handler like the following:
`self.handler = ArithmeticScoringHandler(normalized=False)`